tools/integrations/manage.sh

```diff
@@ -217,19 +220,43 @@ cmd_publish() {
   if [[ "airbyte/normalization" == "${image_name}" ]]; then
     echo "Publishing normalization images (version: $versioned_image)"
     GIT_REVISION=$(git rev-parse HEAD)
-    VERSION=$image_version GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml build
-    VERSION=$image_version GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml push
-    VERSION=latest         GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml build
-    VERSION=latest         GIT_REVISION=$GIT_REVISION docker-compose -f airbyte-integrations/bases/base-normalization/docker-compose.build.yaml push
-  else
-    docker tag "$image_name:dev" "$versioned_image"
-    docker tag "$image_name:dev" "$latest_image"
 
-    echo "Publishing new version ($versioned_image)"
-    docker push "$versioned_image"
-    docker push "$latest_image"
+    # Note: "buildx bake" needs to be run within the directory
+    local original_pwd=$PWD
+    cd airbyte-integrations/bases/base-normalization
+
+    VERSION=$image_version GIT_REVISION=$GIT_REVISION docker buildx bake \
+      --set "*.platform=$build_arch"                                     \
+      -f docker-compose.build.yaml                                       \
+      --push
+
+    VERSION=latest GIT_REVISION=$GIT_REVISION docker buildx bake \
+      --set "*.platform=$build_arch"                             \
+      -f docker-compose.build.yaml                               \
+      --push
+
+    cd $original_pwd
+  else
+    # We have to go arch-by-arch locally (see https://github.com/docker/buildx/issues/59 for more info) due to our base images (e.g. airbyte-integrations/bases/base-java)
+    # Alternative local approach @ https://github.com/docker/buildx/issues/301#issuecomment-755164475
+
+    for arch in $(echo $build_arch | sed "s/,/ /g")
+    do
+      echo "building base images for $arch"
+      docker buildx build -t airbyte/integration-base:dev --platform $arch --load airbyte-integrations/bases/base
+      docker buildx build -t airbyte/integration-base-java:dev --platform $arch --load airbyte-integrations/bases/base-java
+
+      local arch_versioned_image=$image_name:`echo $arch | sed "s/\//-/g"`-$image_version
```

Comment:
Comment on why the `sed` part is necessary?

Reply:
docker tags dont like backslash I assume?

Reply:
yep - exactly.  So the tags end up looking like `linux-arm64-0.0.1` and `linux-amd64-0.0.1` - which you can see a demo of here https://hub.docker.com/repository/docker/evantahler/destination-postgres/tags